### ISSUE TITLE
m3u8 search engine where ?

### ISSUE THREAD

**Issue by spotanjo3** [iptv-org/iptv#13651]
### Channel ID

none

### Notes (optional)

I remember that you have search engine for m3u8 links. I forgot. 

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by PopeyeTheSai10r** [comment]
https://iptv-org.github.io/ 
Is this what you asking for?

**Comment by PopeyeTheSai10r** [comment]
https://github.com/orgs/iptv-org/discussions is more appropriate for questions like this.

**Comment by spotanjo3** [comment]
Yes, thank you. Bookmark it this time. And I know that I should do this in discussions thread. Thank you!
